backend/games/word_guesser/ClassicWordle.py

```diff
@@ -0,0 +1,137 @@
+from random import randint
+from typing import Dict, List
+
+
+class ClassicWordle:
+    # maximum number of guesses
+    MAX_GUESSES = 6
+    WORD_LIST = [
+        "adieu",
+        "odium",
+        "shade",
+        "resin",
+        "alert",
+        "haunt",
+        "orate",
+        "media",
+        "blind",
+        "route",
+        "audio",
+        "pause",
+        "alien",
+        "canoe",
+        "plane",
+        "rouse",
+        "fraud",
+        "atone",
+        "raise",
+        "minor",
+    ]
+
+    def __init__(self):
+        # represents the state of the game, index 0 will hold the "goal word"
+        game_board = List[str]
+        guesses = {"guess1": "", "results": []}
+        goal_word = ""
+        guess_count = 0
+
+    # clears any previous activity and randomizes a word to initialize the list
+    def start_game(self) -> List[str]:
+        # clear past game data
+        guesses = {"guess1": "", "results1": []}
+        game_board = [""]
+        goal_word = str
+        guess_count = 0
+
+        # randomize next word
+        new_word_index = randint(0, 19)  # nosec
+        goal_word = WORD_LIST[new_word_index]
```

Comment:
This will cause a `mypy` error because it does not find `WORD_LIST` in scope. Since you have `WORD_LIST` as a class variable, try to access it by `ClassicWordle.WORD_LIST`.

Or you can make it a global variable in this file, not just the class.